packages/terra-application-layout/docs/responsiveDesign.md

```diff
@@ -0,0 +1,32 @@
+# Terra Application Layout
+
+## Responsive Design
+
+The ApplicationLayout has two rendering modes: `standard` and `compact`.
+  - The `standard` rendering occurs at `medium`, `large`, and `huge` breakpoints.
+  - The `compact` rendering occurs at `tiny` and `small` breakpoints.
+
+### Header
+
+The ApplicationLayout header has a number of changes between its `standard` and `compact` rendering modes.
```

Comment:
> number of changes

This wording seems odd. I would maybe do something like `The ApplicationLayout header handles its props different in `standard` mode compared to `compact` mode.

Reply:
I'll blame that one on too little caffeine.